editing/include/tests.js

```diff
@@ -5582,7 +5582,7 @@ function runConformanceTest(browserTest) {
 
             assert_equals(document.execCommand(browserTest[1][i][0], false, browserTest[1][i][1]),
                 expectedExecCommandReturnValues[i]);
-        }, testName + ": execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
+        }, testName + ": command #" + (i+1) + " execCommand(" + format_value(browserTest[1][i][0]) + ", false, " + format_value(browserTest[1][i][1]) + ") return value");
```

Comment:
> Sequence numbers are generally not a great idea as they have a tendency to
> change.

True, but the command sequence numbers are more than just test implementation
details--they are an integral part of each test. In other words, if the
sequence of a given test's commands were to change, then that would be a
distinct test.

I wasn't aware of any expectations about the stability of sub-test names. Is
that detail significant to consumers?

> It would be better to either only use this iff there would be a dupe, or to
> ensure that there is never a dupe.

We can't ensure there this is never a duplicate because some tests require
repeated applications of the same command. The first option is do-able, but I
don't know if the complexity is warranted (see above).

Then again, the surrounding test generation logic is already *very* involved,
so a few more branches couldn't hurt. I've said my peace--if you'd still be
more comfortable with "conditionally-specified" sequence numbers, then say the
word and I'll patch it in.